### PR TITLE
adding cookbook examples

### PR DESCRIPTION
**Pull request by iruoma**
This PR contributes to https://github.com/ploomber/ploomber/issues/317

### CODE REVIEW

**Comment by iruoma** [comment]
I don't see a link to a complete example for ['Debugging'](file:///C:/Users/Iruoma/Documents/GitHub/ploomber/doc/_build/html/cookbook/debugging.html) in 'https://github.com/ploomber/projects/tree/master/cookbook'. Should I be looking elsewhere?

**Comment by lfunderburk** [comment]
Thank you @iruoma!
@edblancas it seems that there wasn't a folder for Debugging in the cookbook repo. Do you know if the folder is stored elsewhere?

**Comment by edublancas** [comment]
not every section in the docs has a relevant example in the cookbook so that's fine :) just add the ones that are related

File: doc/cookbook/db-config.rst [hunk 1]
```diff
@@ -149,4 +149,6 @@ Configuring the client in ``pipeline.yaml``
 -------------------------------------------
 
 Check out the :doc:`../get-started/sql-pipeline` to learn how to configure
-the database client in your ``pipeline.yaml`` file.
\ No newline at end of file
+the database client in your ``pipeline.yaml`` file. 
+
+You can see a complete 'example <https://github.com/ploomber/projects/tree/master/cookbook/sql-dump>'_.
```
Comment (edublancas) [inline doc/cookbook/db-config.rst:154]: thanks for adding this. i think it'd make sense to create a new section, something like this:

```
SQL Examples
---------------

To see some examples using SQL connections, see this:

[list of relevant links]

```

**Review by edublancas — CHANGES_REQUESTED** [review]

File: doc/cookbook/db-config.rst (hunk 1 above)
Comment (iruoma) [reply doc/cookbook/db-config.rst:154]: sure will do that

Comment (edublancas) [reply doc/cookbook/db-config.rst:154]: here are two other links you can include: https://github.com/ploomber/projects/tree/master/templates/etl

https://github.com/ploomber/projects/tree/master/templates/spec-api-sql

Comment (edublancas) [reply doc/cookbook/db-config.rst:154]: also, check the links syntax: 

```
`Link text <link URL>`_
```

Comment (iruoma) [reply doc/cookbook/db-config.rst:154]: are the 2 additional links also for the 'Database Configuration' page?

Comment (edublancas) [reply doc/cookbook/db-config.rst:154]: yep

File: doc/cookbook/db-config.rst [hunk 2]
```diff
@@ -149,4 +149,11 @@ Configuring the client in ``pipeline.yaml``
...
-the database client in your ``pipeline.yaml`` file.
\ No newline at end of file
+the database client in your ``pipeline.yaml`` file. 
+
+SQL Examples
+---------------
+To see some examples using SQL connections, see this:
+1. A complete `example: <https://github.com/ploomber/projects/tree/master/cookbook/sql-dump>`_.
+2. `https://github.com/ploomber/projects/tree/master/templates/etl`_.
+3. `https://github.com/ploomber/projects/tree/master/templates/spec-api-sql`_.
```
Comment (edublancas) [inline doc/cookbook/db-config.rst:159]: awesome! this looks better. can you update the text descriptions:

1. A short example that dumps data
2. A complete ETL pipeline
3. A SQL pipeline

**Review by edublancas — CHANGES_REQUESTED** [review]

File: doc/cookbook/db-config.rst (hunk 2 above)
Comment (iruoma) [reply doc/cookbook/db-config.rst:159]: yes sure!

Comment (iruoma) [reply doc/cookbook/db-config.rst]: done, thanks!

**Comment by edublancas** [comment]
thanks for your contribution!

**Pull request merged**